### DISCUSSION TITLE
u8g2 - upsizing font scale

### DISCUSSION THREAD

**Discussion by jokokiswan** [openLuat/LuatOS#86] [Q&A] [answered]
how to up size available glyphfont? thank you

        u8g2.SetFont(u8g2.font_unifont_t_symbols);
        u8g2.DrawGlyph(32, 60,9754)

got very small lcd icon result

**Comment by chenxuuu** [comment, marked as answer]
this depends on the `font_unifont_t_symbols` font size

you may need to add your own font...  or just use a picture instead
